lib/kontena/plugin/cloud/token/create_command.rb

```diff
@@ -0,0 +1,14 @@
+class Kontena::Plugin::Cloud::Token::CreateCommand < Kontena::Command
+  include Kontena::Cli::Common
+
+  requires_current_account_token
+
+  parameter "[NAME]", "Description for the personal access token"
+
+  def execute
+    self.name = prompt.ask("Name:") unless self.name
```

Comment:
this by the way can be done like:

```ruby
def default_name
  prompt.ask("Name:")
end
```

Reply:
Cool. Have used that somewhere else but already forgot that.